NSS-Spontaneous-Spoonbills/Sprint2
# Db templates

## Link to Ticket

#9 
Closes #
#9 
## Description of Proposed Changes
user can add training programs
## Steps to Test


```sh
git fetch --all
git checkout db_templates
```

Open in the browser, click training programs, create a new program, enter a program of your choosing.
After creating a program go back to training program list and verify that your program is listed.
